app/models/Assessment.scala

```diff
@@ -51,9 +60,52 @@ object Assessment {
       PropertyAddress.fromString(valuationHistory.address),
       valuationHistory.billingAuthorityReference,
       valuationHistory.currentFromDate,
-      valuationHistory.currentToDate,
-      capacityDeclaration
+      valuationHistory.currentToDate
+    )
+  }
+
+  def fromValuationHistory(valuationHistory: ValuationHistory, authorisationId: Long) = {
+    Assessment(
+      authorisationId,
+      valuationHistory.asstRef,
+      valuationHistory.listYear,
+      valuationHistory.uarn,
+      valuationHistory.effectiveDate.getOrElse(LocalDate.now()),
+      valuationHistory.rateableValue.map {d => d.longValue()},
```

Comment:
be careful of longValue, there is rounding involved in that.

Reply:
Are you happy for me to change rateableValue to a BigDecimal on the Frontend? Again this is used in DvrController.alreadySubmittedDetailedValuationRequest

Reply:
Left as is for the time being. Is it possible to have pence the the rateableValue?